docs/source/components/nodes/object_tracker.rst

```diff
@@ -53,10 +53,18 @@ Zero term tracking performs object association, which means that it does not con
 Object association would mean that detected objects from an external detector are mapped with tracked objects which has been detected and is being tracked from
 previous frames.
 
-Supported object tracker types:
+Supported object tracker types
+##############################
 
 - :code:`ZERO_TERM_COLOR_HISTOGRAM`: Utilizes position, shape and input image information such as RGB histogram to perform object tracking.
 - :code:`ZERO_TERM_IMAGELESS`: Only utilizes rectangular shape of detected object and position information for object tracking. It does not use color information of tracking objects. It achieves higher throughput than ZERO_TERM_COLOR_HISTOGRAM. User needs to consider the trade-off between throughput and accuracy when choosing the object tracker type.
+- :code:`SHORT_TERM_IMAGELESS`: Short-term tracking allows to track objects between frames.
```

Comment:
Zero-term tracking assigns unique object IDs and requires object detection to run on every frame. Short-term tracking allows to track objects between frames, thereby reducing the need to run object detection on each frame.

This might help:
https://github.com/openvinotoolkit/dlstreamer_gst/wiki/Object-tracking
This looks like the equivalent of what is ported to device.